src/installer/pkg/projects/Directory.Build.targets

```diff
@@ -15,11 +15,11 @@
   </PropertyGroup>
 
   <!--
-    For mobile targets, we do not need the host, so allow depproj's to opt 
-    out and skip building. 
+    Mono does not need some pkgproj (no host on mobile, no crossgen for desktop), so allow projects to 
+    opt out.
   -->
-  <Target Name="MobileSkipBuildProps" 
-          Condition="'$(SkipBuildOnRuntimePackOnlyOS)' == 'true' and ('$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS' or '$(TargetOS)' == 'Android')"
+  <Target Name="MonoSkipBuildProps" 
+          Condition="('$(SkipBuildOnRuntimePackOnlyOS)' == 'true' and '$(TargetsMobile)' == 'true') or '$(SkipOnMono)' == 'true'"
```

Comment:
Should we change this condition to `'$(SkipBuildOnRuntimePackOnlyOS)' == 'true' or '$(RuntimeFlavor)' == 'Mono'` ?

Reply:
Yeah, that's what I'm going to change it to.